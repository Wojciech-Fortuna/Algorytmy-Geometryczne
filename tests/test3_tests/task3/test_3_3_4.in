0.03877999059615596 -0.025073528383292402
0.011501764789704341 -0.007212008775449252
0.038336442209059185 0.0062683833814135
0.015493700273575314 0.013008579459844863
0.04121950672518822 0.027500001028472332
0.011501764789704341 0.050416667695139
-0.039506299726424686 0.007279412793178203
-0.022873235210295655 -0.009908087206821806
-0.04527242875868275 -0.03080269504995907
-0.00047404166190855024 -0.041924018579370836
