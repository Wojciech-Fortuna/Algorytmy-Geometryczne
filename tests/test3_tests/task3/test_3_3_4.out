3 5
3 6
2 6
1 6
7 1
0 7
8 0